Cargo.toml

```diff
@@ -1,5 +1,6 @@
 [workspace]
 members = ["crates/*"]
+resolver = "2"
```

Comment:
I think this is the new and improved resolver -- https://doc.rust-lang.org/cargo/reference/resolver.html#feature-resolver-version-2